lib/src/core/bloc/bloc.dart

```diff
@@ -0,0 +1,6 @@
+abstract class Bloc{
```

Comment:
This is a IBloc right? And also i think you missed the format here

Reply:
Yes it is! I changed the name and formatted it